src/client/refresh/gl3/gl3_sdl.c

```diff
@@ -341,3 +341,27 @@ void GL3_ShutdownContext()
 		}
 	}
 }
+
+/*
+ * Detect default desktop mode
+ */
+qboolean
+GL3_GetDesktopMode(int *pwidth, int *pheight)
+{
+	// Declare display mode structure to be filled in.
+	SDL_DisplayMode current;
+
+	// We can't get desktop where we start, so use first desktop
+	if(SDL_GetDesktopDisplayMode(0, &current) != 0)
```

Comment:
can't you use https://wiki.libsdl.org/SDL_GetWindowDisplayIndex to get the desktop the window is currently on?

Reply:
Window is already destroyed when code run GetDesktopMode, I have moved detection code to client/vid/glimp_sdl.c as more reliable place for get window (f0dc7bd).

Reply:
@DanielGibson I have also added code for save window position, in other case window will always recreated/moved to first desktop.